lib/pymedphys/docs/README.rst

```diff
@@ -197,8 +197,8 @@ Past contributors
 
 .. _`Release Notes`: ./CHANGELOG.md
 
-.. _`App Users Guide`: https://docs.pymedphys.com/app/index.rst
-.. _`Library Users Guide`: https://docs.pymedphys.com/lib/index.rst
-.. _`CLI Users Guide`: https://docs.pymedphys.com/cli/index.rst
-.. _`Contributors Guide`: https://docs.pymedphys.com/contrib/index.rst
-.. _`General`: https://docs.pymedphys.com/general/index.rst
+.. _`App Users Guide`: https://docs.pymedphys.com/app/index.html
```

Comment:
Sorry about this! I kinda blindly copied from [this line](https://github.com/pymedphys/pymedphys/pull/1350/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fL283).

Reply:
No need to be sorry 🙂 I reviewed it too 🙂.

Bound to be a few hiccups when this level of re-structuring is undergone.